sql/updates/latest-dev.sql

```diff
@@ -309,8 +309,19 @@ SELECT materialization_id, BIGINT '-9223372036854775808', watermark, 92233720368
 FROM _timescaledb_catalog.continuous_aggs_completed_threshold;
 -- Also handle continuous aggs that have never been run
 INSERT INTO _timescaledb_catalog.continuous_aggs_materialization_invalidation_log
-SELECT (SELECT mat_hypertable_id FROM _timescaledb_catalog.continuous_agg EXCEPT SELECT materialization_id FROM _timescaledb_catalog.continuous_aggs_completed_threshold), 
--9223372036854775808, -9223372036854775808, 9223372036854775807;
+SELECT unrun_cagg.id, -9223372036854775808, -9223372036854775808, 9223372036854775807 FROM
+(SELECT mat_hypertable_id id FROM _timescaledb_catalog.continuous_agg EXCEPT SELECT materialization_id FROM _timescaledb_catalog.continuous_aggs_completed_threshold) as unrun_cagg;
+
+-- Also add an invalidation from [-infinity, ignore_invaliation_older_than] to cover any missed invalidations
+-- For NULL or inf ignore_invalidations_older_than, use julian 0 for consistency with 2.0
+INSERT INTO _timescaledb_catalog.continuous_aggs_materialization_invalidation_log
+SELECT cagg.mat_hypertable_id, BIGINT '-9223372036854775808', BIGINT '-9223372036854775808', 
```

Comment:
I think this should be added only if ignore_invalidation_older_than is not the default value and not the other case.
Could you explain how we get the value at line 321?  ignore_invalidation_older_than could be set for an integer table. In this case what does it mean to subtract from now() ( line 322)?

Reply:
This is the invalidation that 2.0 adds to every new cagg by default.  I'm not entirely sure what the reasoning behind this was, but I figured it would be better to match the 2.0 invalidation here.  I'm not sure what you're asking about the subtraction from now(), as that only happens in the other case.